<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>196</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textUserId">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>40</y>
     <width>271</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textPasswd">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>80</y>
     <width>271</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>4</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Server IP:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>4</y>
     <width>30</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>40</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>86</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textSvrIp">
   <property name="geometry">
    <rect>
     <x>76</x>
     <y>6</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textSvrPort">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>6</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
